sc/social/like/tests/test_plugin_facebook.py

```diff
@@ -132,130 +131,27 @@ def test_privacy_plugin_view_html(self):
         html = view.link()
         self.assertIn('Share on Facebook', html)
 
-    # FIXME: we need to rethink this feature
-    @unittest.skipIf(IS_PLONE_5, 'Metadata viewlet is disabled in Plone 5')
-    def test_plugin_view_metadata(self):
+    def test_plugin_view_javascript(self):
         plugin = self.plugin
         portal = self.portal
         plugin_view = plugin.view()
         view = portal.restrictedTraverse(plugin_view)
 
-        metadata = view.metadata()
-        self.assertIn('og:site_name', metadata)
-
-        # At root, use site logo
-        image_url = view.image_url()
-        self.assertIn('logo.png', image_url)
-
-        # At root, use website type
-        og_type = view.type()
-        self.assertIn('website', og_type)
-
-    def test_plugin_view_document(self):
-        plugin = self.plugin
-        document = self.document
-        portal = self.portal
-
-        plugin_view = plugin.view()
-        view = document.restrictedTraverse(plugin_view)
-
-        # At document, use site logo
-        image_url = view.image_url()
-        self.assertIn('logo.png', image_url)
-
-        # At document, use article type
-        og_type = view.type()
-        self.assertIn('article', og_type)
-
-        # At document, default page of portal, use website type
-        portal.setDefaultPage(document.id)
-        og_type = view.type()
-        self.assertIn('website', og_type)
-
-    def test_plugin_view_image(self):
```

Comment:
These tests are not needed anymore since we are using **scale large** by default for all plugins.